packages/rocketchat-lib/server/methods/sendInvitationSMS.js

```diff
@@ -0,0 +1,64 @@
+import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/tap:i18n';
+import { check } from 'meteor/check';
+import _ from 'underscore';
+
+Meteor.methods({
+	sendInvitationSMS(phones) {
+		const twilioService = RocketChat.SMS.getService('twilio');
+		if (!RocketChat.SMS.enabled || !twilioService) {
+			throw new Meteor.Error('error-twilio-not-active', 'Twilio service not active', {
+				method: 'sendInvitationSMS',
+			});
+		}
+
+		const messageFrom = RocketChat.settings.get('Invitation_SMS_Twilio_From');
+		if (!twilioService.accountSid || ! twilioService.authToken || !messageFrom) {
+			throw new Meteor.Error('error-twilio-not-configured', 'Twilio service not configured', {
+				method: 'sendInvitationSMS',
+			});
+		}
+
+		check(phones, [String]);
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+				method: 'sendInvitationSMS',
+			});
+		}
+
+		// to be replaced by a seperate permission specific to SMS later
+		if (!RocketChat.authz.hasPermission(Meteor.userId(), 'bulk-register-user')) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', {
+				method: 'sendInvitationSMS',
+			});
+		}
+		const phonePattern = /^\+?[1-9]\d{1,14}$/;
+		const validPhones = _.compact(_.map(phones, function(phone) {
+			if (phonePattern.test(phone)) {
+				return phone;
+			}
+		}));
+		const user = Meteor.user();
+		let body;
+		if (RocketChat.settings.get('Invitation_SMS_Customized')) {
+			body = RocketChat.settings.get('Invitation_SMS_Customized_Body');
+		} else {
+			const lng = user.language || RocketChat.settings.get('language') || 'en';
```

Comment:
Can you clarify this for me please?  

Is "user.language" in this case the user who is sending the invitation?  If so, is there some condition when we won't be getting that flag from the user, so that we need these '||' statements?

Is RocketChat.settings.get('language') the language that the admin uses on the RC server?  (Which we know will be English because we are going to be the admins)?

Reply:
I replicated it from the RC code here. https://github.com/RocketChat/Rocket.Chat/blob/14f47007aec90aee4ed810d1250bc71f328df7f0/packages/rocketchat-livechat/messageTypes.js#L37

I will checkout for the cases (if they exist) when the first two will not be set.

Reply:
Interesting.....  the file you linked to is "live chat", and the user will be on a web browser I believe.... so we can likely expect that we'll get "user.language", because web browsers do that by default.  We should verify what this looks like when it's coming from the android client, because we most definitely want to send the invite in the native language.

Reply:
Make a note of this and track for future PR?

Reply:
Sounds good.